src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -261,12 +261,16 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
         }
 
         private static bool BackendSupportsCustomCertificateHandling =>
-            RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ||
+#if unix
```

Comment:
We've tried hard to avoid platform-based ifdefs like this.  Why is this necessary?